src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
```

Comment:
Ensure that the `variables` file contains all the necessary variable definitions (e.g., `$main-text-color`, `$card-main-color`, `$nav-text-color`, `$nav-text-color-active`, `$star-bg`, `$star-bg-active`) used across the imported SCSS files to prevent any compilation errors.